politeiawww/api/v1/api.md

```diff
@@ -872,7 +850,23 @@ SHALL observe.
 
 **Params:** none
 
-**Results:** see below
+**Results:** 
+
+| | Type | Description |
+|-|-|-|
+| passwordminchars | integer | minimum number of characters accepted for user passwords |
```

Comment:
I think this changed after you opened the PR, it's now called `minpasswordlength`. Could you also add `minusernamelength`, `maxusernamelength`, and `usernamesupportedchars`?